ftest.sh

```diff
@@ -151,14 +151,6 @@ if [ \"\${HOSTNAME%%%%.*}\" != \"${nodes[0]}\" ]; then
             sudo mkdir -p /mnt/daos
         fi
     fi
-
```

Comment:
I don't think this was the intent of Brian's comment.  This code is needed for all the other tests.  The `ServerCommand.clean_server()` should not be unmounting and mounting the mount point.

Reply:
clean_server is only unmounting. The mounting of the /mnt/daos directory is handled by the server. From what Mike mentioned to me, if the server does not find the directory it will need a storage format, so because of that I think we should keep it in there...